feat: ✨ LS_COLORS generated with vivid

tkapias opened the pull request:
I generated a theme for LS_COLORS with vivid as suggested in #56. 

Use the theme yml file with vivid or export the variable included in the .bashrc file in your shell.

tkapias commented:
![image](https://github.com/user-attachments/assets/4c9e2df8-281e-45f0-a9b9-c0d8a8773e8f)
![image](https://github.com/user-attachments/assets/c8f09798-2e60-4f06-8cf3-ac46c5ee287a)

fenetikm reviewed (changes requested):
Thanks - just one little thing I think.

tkapias commented:
I changed the name of the file and updated the colors, it's much better.

I did not put a comment, but LS_COLORS is useful for many commands like ls, tree, find, du, lsd... I let you decide if it needs to be commented.

![image](https://github.com/user-attachments/assets/c79712d4-a2b6-4c31-b6f3-d65e6743b7ef)
![image](https://github.com/user-attachments/assets/f68426a6-bc3e-456c-aea0-8ffdbf5d83a4)
![image](https://github.com/user-attachments/assets/6a75f0d8-e097-462b-a819-8fea5c6d404f)

tkapias commented on vivid/.bashrc:
```diff
@@ -0,0 +1,2 @@
+export LS_COLORS='*~=0;38;2;54;54;58:bd=0;38;2;2;2;33;48;2;153;164;188:ca=0;38;2;2;2;33;48;2;54;54;58:cd=0;38;2;2;2;33;48;2;153;164;188:di=1;38;2;255;211;146:do=0;38;2;2;2;33;48;2;255;211;146:ex=1;38;2;255;142;120:fi=0:ln=3;38;2;191;218;255:mh=0:mi=0;38;2;191;218;255:no=0;38;2;180;180;185:or=3;38;2;191;218;255;48;2;255;54;0:ow=1;38;2;255;211;146;48;2;99;81;150:pi=0;38;2;2;2;33;48;2;255;176;123:rs ...
```
Done

fenetikm commented:
LGTM, cheers 🍻
